api/api/serializers/media_serializers.py

```diff
@@ -260,6 +272,23 @@ def validate_unstable__include_sensitive_results(
 
         return self.initial_data.get("mature") or value
 
+    def validate_internal__index(self, value):
+        """
+        Check whether the given index name is a valid index or alias. However,
+        for unauthenticated requests, no check is performed and ``None`` is
+        returned immediately.
+
+        :param value: the provided index name to check
+        :return: ``None`` if request is anonymous, the provided name if it is valid
+        :raise: ``serializers.ValidationError`` if not anonymous and invalid index name
+        """
+
+        if self.is_request_anonymous():
+            return None
```

Comment:
Should this be restricted to a specific OAuth scope, one that people cannot get by merely registering a plain API key?